index.js

```diff
@@ -299,14 +299,14 @@ class Applesign {
         delete entMacho['com.apple.developer.associated-domains'];
         delete entMacho['com.apple.developer.team-identifier'];
       }
+    }    
+
+    if( typeof this.config.withGetTaskAllow != 'undefined' ) {
```

Comment:
Please use semistandard to indent the code properly (npm run indent)

Reply:
Use !== instead if !=

Reply:
These are addressed now. Please check.

Reply:
@radare Can you review the new changes and approve the PR?